osquery/tables/specs/x/fde_status.table

```diff
@@ -0,0 +1,11 @@
+table_name("fde_status")
```

Comment:
Food for thought: I'm not 100% sold on this name, what do you think of: `disk_encryption` (doesn't fit with the mostly-plural existing names), `disk_encryption_status`, `disk_encryption_details`, `fde_details`? The reason I bring it up is there are a few instances where our not-so-standard acronyms may add confusion: `ad_config` (what does AD mean?), and `alf`, `application_firewall` would mean more.

It up to you. :) In this case "fde" is more common, so whatever name you think is most appropriate.